javascript/code-challenges/fifoAnimalShelter/fifo-animal-shelter.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const Node = require('../node.js');
+
+class AnimalQueue {
+  constructor() {
+    this.storage = nul
+    l;
```

Comment:
This is causing issues because you accidentally spaced the last l on a separate line